src/tests/contract_test/wallet_test.cljs

```diff
@@ -65,3 +61,27 @@
     (fn []
       (p/let [response (contract-utils/call-rpc "wallet_getEthereumChains")]
         (assert-ethereum-chains response)))))
+
+(defn assert-wallet-tokens
+  [tokens]
+  (let [flattened-tokens (mapcat val tokens)]
+    (doseq [token flattened-tokens]
+      (is (not-empty (:symbol token)))
+      (is (:decimals token))
+      (is (contains? token :balancesPerChain))
```

Comment:
A simple one up the top of my mind:
```suggestion
      (is (contains? token :balancesPerChain))
      (is (contains? token :marketValuesPerCurrency))
      (is (contains? (:marketValuesPerCurrency token) :usd))
```

Reply:
why `:marketValuesPerCurrency` is important to the tests? (nor arguing, just curious)

Reply:
make sense. Thank you! I;'ve included